src/Tribe/iCal.php

```diff
@@ -420,7 +420,7 @@ public function generate_ical_feed( $post = null, $echo = true ) {
 
 		$content .= 'X-WR-CALNAME:' . $x_wr_calname . "\r\n";
 		$content .= 'X-ORIGINAL-URL:' . $blogHome . "\r\n";
-		$content .= 'X-WR-CALDESC:Events for ' . $blogName . "\r\n";
+		$content .= 'X-WR-CALDESC:' . esc_html__( 'Events for ', 'the-events-calendar' ) . $blogName . "\r\n";
```

Comment:
Hey there @gafderks - love the idea here - but might it be better to use substitution via `sprintf()` and to also provide some context - something like this?

```php
sprintf( esc_html_x( 'Events for %s', 'iCal feed description', 'the-events-calendar' ), $blogname )
```

Adding context would make it easier for translators to target this string specifically and using the `%s` placeholder would allow us to cleanly support languages with a different word order.

Reply:
Thanks for the feedback! I have implemented these suggestions.